src/main/java/br/com/mercadolivre/loja/user/User.java

```diff
@@ -15,6 +17,8 @@
     private String login;
     private String password;
 
+
+    @Deprecated
     public User() {}
 
     public User(String login, String password) {
```

Comment:
Pode usar o construtor default aqui.

```suggestion
    User(String login, String password) { }
```